test/lib/bridge/content_test.rb

```diff
@@ -0,0 +1,100 @@
+require 'test_helper'
+
+class Huboard::ContentsTest < ActiveSupport::TestCase
+  sut = Huboard::Contents
+
+  class MockBridge
+    include Huboard::Contents
+  end
+  bridge = MockBridge.new
+
+
+  describe 'issue_template' do
+
+    before(:each) do
+      @mock_gh_instance = mock()
+      bridge.stubs(:gh).returns(@mock_gh_instance)
+    end
+
+    describe 'the template exists' do
+      file_list = [
+        {'name' => 'ISSUE_TEMPLATE.txt', 'path' => 'ISSUE_TEMPLATE.txt'},
+        {'name' => 'application.js'},
+        {'name' => 'huboard_is_awesome.rb'}
+      ]
+
+      it 'looks for a template in the root of the repo' do
+        @mock_gh_instance.stubs(:contents).returns(file_list)
+        @mock_gh_instance.expects(:contents).with('').once
+        @mock_gh_instance.expects(:contents).with('ISSUE_TEMPLATE.txt').returns(file_list[0])
+
+        assert_equal file_list[0], bridge.issue_template
+      end
+
+      it 'looks for a template in the .github directory' do
+        @mock_gh_instance.expects(:contents).with('').returns []
```

Comment:
It occurs to me that this would include at least the `.github` folder, which we could check for in the root response before making the directory listing.